src/graph/context/ast/CypherAstContext.h

```diff
@@ -210,8 +215,8 @@ struct NodeContext final : PatternContext {
   QueryContext* qctx;
   Expression* bindFilter;
   GraphSpaceID spaceId;
-  NodeInfo* info{nullptr};
```

Comment:
Why remove this initialization?

Reply:
will be initialized in the initialization list